suites/upgrade/infernalis-x/parallel/5-final-workload/rados-snaps-few-objects.yaml

```diff
@@ -0,0 +1,17 @@
+meta:
+- desc: |
+   add test details here
```

Comment:
randomized correctness test for rados operations on a replicated pool with snapshots